src/object/cli_obj.c

```diff
@@ -895,7 +935,6 @@ obj_rw_req_reassemb(struct dc_object *obj, daos_obj_rw_t *args,
 		return 0;
 
 	if (!obj_auxi->req_reasbed) {
-		obj_auxi->is_ec_obj = 1;
```

Comment:
why remove it? seems that flag is still used and not set anywhere?

Reply:
oh, moved to obj_task_init_common